cypress/integration/Case07_Send_money.js

```diff
@@ -15,16 +15,41 @@ describe('Test case 7: Ability to send money', () => {
       LoginPage.mnemonicsInput.type(mnemonic)
       LoginPage.recoverWalletButton.click()
       LoginPage.yayButton.click()
-      HomePage.claimButton.click().invoke('text').then(text => {
-        cy.log(text)
-        if (text == 'Queue') {
-          const urlRequest = Cypress.env('REACT_APP_SERVER_URL')
-          const bodyPass = Cypress.env('CYPRESS_GUNDB_PASSWORD')
-          cy.request('POST', urlRequest + '/admin/queue', [{ password: bodyPass, allow: 1 }])
-        }
+      HomePage.claimButton.click()
+      cy.wait(2000) // wait for a pop-up to appear confirming the decision in the queue
+      cy.get('#root')
+        .find('[role="button"]')
+        .its('length')
+        .then(res => {
+          cy.log(res)
+          if (res == 4) {
+            cy.get('[role="button"]')
+              .contains(/OK, I’ll WAIT/i)
+              .click()
+          }
+          if (res != 2) {
+            HomePage.claimButton.invoke('text').then(text => {
+              cy.log(text)
+              if (text == 'Queue') {
+                const urlRequest = Cypress.env('REACT_APP_SERVER_URL')
+                const bodyPass = Cypress.env('GUNDB_PASSWORD')
+                cy.request('POST', urlRequest + '/admin/queue', { password: bodyPass, allow: 0 }).then(response => {
+                  expect(response.body).to.have.property('stillPending')
+                })
+                cy.reload()
+                cy.contains('Welcome to GoodDollar!').should('be.visible')
+                HomePage.claimButton.should('be.visible')
+                HomePage.claimButton.click()
+              }
+            })
+          }
+        })
 
       SendMoneyPage.dailyClaimText.should('be.visible')
       SendMoneyPage.claimButton.click()
+      cy.get('[role="button"]')
```

Comment:
which one button you testing ? add it to the page object please or at least comment what you're testing exactly

Reply:
`cy.get('[role="button"]')`
`.eq(1)`
`.should('have.attr', 'data-focusable')`
waiting for the button to become active to continue the test

Reply:
for **which** button ? i don't understand. please add at least a comment (of course, to define the button in the page object will be better approach)